04-gas/gas-section-one/gas-section-one.css

```diff
@@ -103,14 +113,13 @@
 .gas-section-one-image-container {
   grid-area: gas-section-one-image-container;
   display: grid;
-  justify-content: center;
-  align-content: center;
+  align-content: start;
```

Comment:
im still not sure about if we should use `justify-content: center` on all the image containers
but i think for now just add it to all of them...
i've been trying to work out the real differences that occur when it is applied and when it is not applied...

Reply:
actually it doesn't matter because the width of images is 100% , so justify content doesn't do any thing but if we change the width of image it will make sense

Reply:
i have added for now